samples/02_power_users_developers/count_the_usage_frequency_of_api_methods.ipynb

```diff
@@ -0,0 +1,986 @@
+{
```

Comment:
<p>Just a thought, does it make more sense to look at the popularity for each module like raster and feature module? For example, some might be interested to see which raster function is the most popular.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/514/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>